postgresql-backend/src/main/java/de/bwaldvogel/mongo/backend/postgresql/PostgresqlBackend.java

```diff
@@ -43,4 +42,8 @@ public Connection getConnection() throws SQLException {
         return dataSource.getConnection();
     }
 
+    @Override
+    public void enableOplog() {
+    }
```

Comment:
perhaps this should throw `UnsupportedOperationException` for Backends that do not support it

Reply:
this was a mistake. By default every backend is going to use the CollectionBackedOplog.